src/components/flexjar/Flexjar.tsx

```diff
@@ -0,0 +1,227 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Alert,
+  BodyShort,
+  Box,
+  Button,
+  ErrorMessage,
+  Label,
+  Radio,
+  RadioGroup,
+  Textarea,
+  UNSAFE_Combobox,
+} from '@navikt/ds-react';
+import {
+  FlexjarFeedbackDTO,
+  useFlexjarFeedback,
+} from '@/data/flexjar/useFlexjarFeedback';
+import { ChevronDownIcon, ChevronUpIcon } from '@navikt/aksel-icons';
+import { defaultErrorTexts } from '@/api/errors';
+import { StoreKey, useLocalStorageState } from '@/hooks/useLocalStorageState';
+import * as Amplitude from '@/utils/amplitude';
+import { EventType } from '@/utils/amplitude';
+
+const texts = {
+  apneKnapp: 'Vi ønsker å lære av deg',
+  anonym: 'Anonym tilbakemelding',
+  alert: 'Ikke skriv inn navn eller andre personopplysninger.',
+  feedbackLabel: (svar: string) =>
+    svar === 'Ja'
+      ? 'Noe annet du bruker Arena til?'
+      : 'Har du noen andre tilbakemeldinger til oss?',
+  feedbackLabelDescription:
+    'Hvis ikke, kan du trykke på "Send tilbakemelding".',
+  send: 'Send tilbakemelding',
+  validation: 'Vennligst velg en tilbakemelding',
+  success: 'Takk for din tilbakemelding!',
+  sporsmal: 'Bruker du Arena til daglig i sykefraværsoppfølgingen?',
+  labelAlternativer: 'Hva bruker du Arena til? (Velg gjerne flere)',
+};
+
+const arenaOptions = [
+  '39-ukers oppgave',
+  '14a-vurdering',
+  'AEV',
+  'Historikk på sykmeldt',
+  'Dialogmelding',
+  'Sende brev fra egen mal',
+  'Friskmelding til arbeidsformidling',
+  'Tiltak',
+  'Be om oppfølgingsplan',
+  'Vurdere § 8-4',
+  'Vurdere § 8-8 manglende medvirkning',
+  'Oppgave fra behandler i sykmeldingen',
+];
+
+function logPageView(side: string) {
+  Amplitude.logEvent({
+    type: EventType.PageView,
+    data: { url: window.location.href, sidetittel: side + ' - med Flexjar' },
+  });
+}
+
+interface FlexjarProps {
+  side: string;
+}
+
+export const Flexjar = ({ side }: FlexjarProps) => {
+  const [isApen, setIsApen] = useState<boolean>(true);
+  const [isValid, setIsValid] = useState<boolean>();
+  const [feedback, setFeedback] = useState<string>();
+  const sendFeedback = useFlexjarFeedback();
```

Comment:
Kanskje flytte denne ned sånn at `useState` ligger samlet - for ryddighetens skyld.